connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -486,12 +440,8 @@ public void testReadEndOffsetsUsingAdmin() throws Exception {
         Map<TopicPartition, Long> endOffsets = new HashMap<>();
         endOffsets.put(TP0, 0L);
         endOffsets.put(TP1, 0L);
-        admin.retryEndOffsets(EasyMock.eq(tps), EasyMock.anyObject(), EasyMock.anyLong());
-        PowerMock.expectLastCall().andReturn(endOffsets).times(1);
-        admin.endOffsets(EasyMock.eq(tps));
-        PowerMock.expectLastCall().andReturn(endOffsets).times(1);
-
-        PowerMock.replayAll();
+        when(admin.retryEndOffsets(eq(tps), any(), anyLong())).thenReturn(endOffsets);
+        when(admin.endOffsets(eq(tps))).thenReturn(endOffsets);
```

Comment:
We're losing out on the existing coverage here since we're no longer ensuring that these methods are called one time only. Can we add an explicit verification for this at the end of the test?